src/py/flwr/common/message.py

```diff
@@ -417,3 +417,28 @@ def _create_reply_metadata(msg: Message, ttl: float) -> Metadata:
         ttl=ttl,
         message_type=msg.metadata.message_type,
     )
+
+
+def check_message_type(message_type: str) -> bool:
```

Comment:
```suggestion
def validate_message_type(message_type: str) -> bool:
```

Given that we use `validate_*` in our `flwr` CLI checks, perhaps `validate_*` is a better function name here.

Reply:
Good idea!